src/actions/posts.js

```diff
@@ -364,6 +364,40 @@ export function editPost(post) {
     });
 }
 
+export function setUnreadPost(userId, postId) {
+    return async (dispatch, getState) => {
+        dispatch({type: ChannelTypes.POST_UNREAD_REQUEST}, getState);
```

Comment:
`dispatch` actually only takes one argument, so you don't need to pass `getState` into it
```suggestion
        dispatch({type: ChannelTypes.POST_UNREAD_REQUEST});
```

Reply:
I shaw dispatch receiving the getState function as parameter in several other places, I thought I was missing something as having it didn't break something (also js doesn't complain about too many arguments...) so I decided to err on the safe side.

I'll remove it